tests/state_test.go

```diff
@@ -15,40 +15,57 @@ import (
 	"github.com/hashicorp/go-hclog"
 )
 
-var (
-	stateTests       = "GeneralStateTests"
-	legacyStateTests = "LegacyTests/Constantinople/GeneralStateTests"
-)
+// Currently used test cases suite version is v10.4.
+// It does not include Merge hardfork test cases.
 
-type stateCase struct {
-	Info        *info                                   `json:"_info"`
-	Env         *env                                    `json:"env"`
-	Pre         map[types.Address]*chain.GenesisAccount `json:"pre"`
-	Post        map[string]postState                    `json:"post"`
-	Transaction *stTransaction                          `json:"transaction"`
-}
+const (
+	stateTests       = "tests/GeneralStateTests"
+	legacyStateTests = "tests/LegacyTests/Constantinople/GeneralStateTests"
+)
 
-var ripemd = types.StringToAddress("0000000000000000000000000000000000000003")
+var (
+	ripemd = types.StringToAddress("0000000000000000000000000000000000000003")
+)
 
-func RunSpecificTest(t *testing.T, file string, c stateCase, name, fork string, index int, p postEntry) {
+func RunSpecificTest(t *testing.T, file string, c testCase, name, fork string, index int, p postEntry) {
 	t.Helper()
 
 	config, ok := Forks[fork]
 	if !ok {
-		t.Fatalf("config %s not found", fork)
+		t.Skipf("%s fork is not supported", fork)
+
+		return
 	}
 
 	env := c.Env.ToEnv(t)
 
-	msg, err := c.Transaction.At(p.Indexes)
+	var baseFee *big.Int
+
+	if config.IsLondon(0) {
+		if c.Env.BaseFee != "" {
+			baseFee = stringToBigIntT(t, c.Env.BaseFee)
+		} else {
+			// Retesteth uses `0x10` for genesis baseFee. Therefore, it defaults to
+			// parent - 2 : 0xa as the basefee for 'this' context.
+			baseFee = big.NewInt(0x0a)
```

Comment:
maybe put this 0x0a value inside some const?

Reply:
Done